koku-metrics-operator/0.9.0/manifests/koku-metrics-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,262 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "koku-metrics-cfg.openshift.io/v1alpha1",
+          "kind": "KokuMetricsConfig",
+          "metadata": {
+            "name": "kokumetricscfg-sample"
+          },
+          "spec": {
+            "authentication": {
+              "type": "token"
+            },
+            "packaging": {
+              "max_size_MB": 100
+            },
+            "prometheus_config": {},
+            "source": {
+              "check_cycle": 1440,
+              "create_source": false,
+              "name": "INSERT-SOURCE-NAME"
+            },
+            "upload": {
+              "upload_cycle": 360,
+              "upload_toggle": true
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Monitoring
+    certified: "false"
+    containerImage: quay.io/project-koku/koku-metrics-operator:v0.9.0
+    createdAt: "2020-12-11T18:28:37Z"
+    description: A Golang-based OpenShift Operator that generates and uploads OpenShift
+      usage metrics to cost management.
+    operators.operatorframework.io/builder: operator-sdk-v0.19.2
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    repository: https://github.com/project-koku/koku-metrics-operator
+    support: Cost Managment
+  name: koku-metrics-operator.v0.9.0
+  namespace: koku-metrics-operator
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KokuMetricsConfig is the Schema for the kokumetricsconfig API
+      kind: KokuMetricsConfig
+      name: kokumetricsconfigs.koku-metrics-cfg.openshift.io
+      version: v1alpha1
+  description: INSERT OPERATOR DESCRIPTION
```

Comment:
We should sed replace this with the csv-description docs

Reply:
Instead of sed replace, I created a python script to make all the necessary file changes.